adamas-app/src/App.scss

```diff
@@ -36,6 +36,11 @@ h3 {
   margin: 0.5rem auto;
 }
 
+li {
+  list-style: url("https://img.icons8.com/metro/26/ffffff/next-generation-badge.png");
```

Comment:
Looks cool, but it's a bit strange to use as bullet points for non-personel data , as it's a personal badge.

Would use it for a crew roster, if we would ever have one.